### DISCUSSION TITLE
Error on jwt custom authentication: Constructing your key on localhost

### DISCUSSION THREAD

**Discussion by ashishspacemishra** [Web3Auth/Web3Auth#1335] [Help]
Hi there I followed the document https://web3auth.io/docs/guides/cognito and have set all parameters correctly. But when I run the app it breaks at 

> Constructing your key on localhost

After which it throws error on UI: 

> Unable to detect login share from the Auth Network. This may be due to slow internet connection. Check your internet speed and try again. If you're using a vpn, please turn it off.

Can someone have a look and tell me what could be wrong here?

<img width="1432" alt="Screenshot 2023-02-16 at 12 39 27 AM" src="https://user-images.githubusercontent.com/97965706/219420031-60c4a9a7-5c81-4f2f-9846-b3b191768eb6.png">

<img width="1394" alt="Screenshot 2023-02-16 at 12 40 43 AM" src="https://user-images.githubusercontent.com/97965706/219420068-4023a3a6-fb59-4d36-aa26-011092ad363c.png">

code snippet:

```
const web3auth = new Web3AuthCore({
    clientId: "", //client-id from web3auth app
    web3AuthNetwork: "mainnet",
    chainConfig: {
        chainNamespace: CHAIN_NAMESPACES.EIP155,
        chainId: "0x89",
        rpcTarget: 'https://polygon-mainnet.infura.io/v3/${REACT_APP_INFURA_ID}',
    },
});

const openloginAdapter = new OpenloginAdapter({
    adapterSettings: {
        uxMode: "popup",
        loginConfig: {
            jwt: {
                name: "AWS Cognito Verifier",
                verifier: "aws-cognito-verifier-gamersxpdapp",
                typeOfLogin: "jwt",
                clientId: "", //jwt-client-id
            },
        },
    },
});
web3auth.configureAdapter(openloginAdapter);

await web3auth.init();

const provider = await web3auth.connectTo(WALLET_ADAPTERS.OPENLOGIN, {
        loginProvider: "jwt",
        extraLoginOptions: {
            domain: "https://overwolf-integration.auth.us-east-1.amazoncognito.com",
            verifierIdField: "email",
            response_type: "token",
            scope: "email profile openid",
        },
    });
```

Thank you in advance for your help. Cheers!

**Comment by ashishspacemishra** [comment]
There are some errors on console as well, which are as following:

> Access to fetch at 'https://overwolf-integration.auth.us-east-1.amazoncognito.com/userinfo' from origin 'https://app.openlogin.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

> Access to fetch at 'https://torus-node.ens.domains/jrpc' from origin 'https://app.openlogin.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

> POST https://torus-node.ens.domains/jrpc net::ERR_FAILED 502

> Error: Could not get result from torus nodes  Error occurred while verifying paramscould not get jwt validation field iss:https://cognito-idp.us-east-1.amazonaws.com/us-east-1_2XjR4VcPE from token &{[7jl0btnoa8nntps3qbco5lie7o] 0xc05f8647a0 0xc05f8647c0 0xc0725d6fd0 0xc0725d7030 <nil> 0xc0725d6fc0 map[at_hash:tPa-NsfCCv0xcwWnxGiPlg auth_time:1.676563057e+09 cognito:username:c5dfed91-d63b-4ce3-a080-fee73c184d10 custom:GDPR-acceptance:0 email:ashishspacemishra@gmail.com email_verified:true name:Ashish nonce:6rh364efwam token_use:id]}
    at D (customAuthError.ts:67:11)
    at a.mounted (Auth.vue?0a0f:126:53)

**Reply by BoE1008** [reply]
Have you solved this problem？

**Reply by khanmubasshir** [reply]
??
